Projects/UOContent/Gumps/AdminGump.cs

```diff
@@ -2147,6 +2150,27 @@ public override void OnResponse(NetState sender, in RelayInfo info)
                                     Shutdown(true, false);
                                     break;
                                 }
+                            case 205:
+                                {
+                                    sender.Mobile.SendMessage("The shard will shutdown in 15 minutes for maintenance.");
+                                    Timer.DelayCall(TimeSpan.FromMinutes(15), () =>
```

Comment:
I am sorry to be nitpicky. Can we do a single timer delay and either call the next timerdelay recursively, or make it an interval with a count (4 or 5, whatever is needed accordingly) and set the Interval in the tick callback. So first one would be delay of 10 minutes and interval of 5 minutes, then after the second tick set interval to 4 minutes, set interval to 1 minute, then shutdown.

I know this is probably confusing, let me know if you need examples, or if this doesn't make sense. Also I am open to a better, easier to read approach.